docs/manuals/source/IntroductionAndTutorial/UpdatingBareos.rst

```diff
@@ -29,59 +429,69 @@ However, they will only be used, when the legacy configuration file does not exi
 See :ref:`section-UpdateToConfigurationSubdirectories` for details and how to migrate to :ref:`section-SubdirectoryConfigurationScheme`.
 
 
+.. _sectionUpdateDatabaseScheme:
+
 Updating the database scheme
-----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Sometimes improvements in Bareos make it necessary to update the database scheme.
 
 
-
 .. warning::
 
    If the Bareos catalog database does not have the current schema, the Bareos Director refuses to start.
 
 
+.. code-block:: shell-session
+   :caption: Shell example of bareos-dir failing to start due to lack of database schema update
+
+   root@host:~# bareos-dir -t
```

Comment:
```suggestion
   root@host:~# su - bareos -s /bin/sh -c "bareos-dir -t"
```
The permission to the database bareos is normally only given to the system user bareos. So, be default, root has no permission to access this database.
Also running bareos-dir as root can result in creating files accessable only as root, which can cause problem by a later regular start (even if I think, this does not happen with the `-t` parameter)

Reply:
was forgotten, otherwise I would have appy my way with bareos-dir -t -u bareos -g bareos ;-)